### PR TITLE
add sepolicy for gralloc default

### PR DESCRIPTION
**Pull request by renchenglei**

### CODE REVIEW

**Comment by renchenglei** [comment]
@feijiang1 & @kwang13 & @ceiba1985, could you help take a review of this changes? Thanks a lot!

**Review by feijiang1 — APPROVED** [review]

File: graphics/mesa/system_server.te [hunk 1]
```diff
@@ -3,3 +3,4 @@ allow system_server platform_app:file { read write };
 allow system_server priv_app:file { read write };
 allow system_server gpu_device:dir r_dir_perms;
 allow system_server sysfs_app_readable:file r_file_perms;
+allow system_server self:process execmem;
```
Comment (ceiba1985) [inline graphics/mesa/system_server.te]: if you already added "PRODUCT_REQUIRES_INSECURE_EXECMEM_FOR_SWIFTSHADER := true", then this line is not needed.

File: graphics/mesa/surfaceflinger.te [hunk 2]
```diff
@@ -26,3 +26,4 @@ allow surfaceflinger hal_graphics_composer_default:file read;
 allow surfaceflinger gpu_device:dir r_dir_perms;
 allow surfaceflinger sysfs_app_readable:file r_file_perms;
 allow surfaceflinger self:process execmem;
+allow surfaceflinger hal_graphics_composer_default:file { getattr open };
```
Comment (ceiba1985) [inline graphics/mesa/surfaceflinger.te]: please merge this line with line 25 (allow surfaceflinger hal_graphics_composer_default:file read;)

**Comment by renchenglei** [comment]
Thanks a lot. Updated the patch, : )

**Review by ceiba1985 — APPROVED** [review]

**Review by kwang13 — APPROVED** [review]

**Review by sgnanase — APPROVED** [review]

**Pull request merged**